### PR TITLE
Fixes multi-tile airlock locations

### LINKED ISSUE #880: Double-airlocks bug
When you construct a multi-tile airlock assembly, moving it around is rather hard - to position it the way you want. Also, when it's finished, even if it's horizontal, it acts like vertical double-airlock, meaning you interact with it when you walk one tile. And when you walk up, you can step through the right tile, as if there was only the sprite.

### PR DESCRIPTION
**Pull request by Dennok**
#880

### CODE REVIEW

File: code/game/objects/structures/door_assembly.dm [hunk 1]
```diff
@@ -282,8 +282,7 @@
 			else
 				path = text2path("/obj/machinery/door/airlock[airlock_type]")
 
-			var/obj/machinery/door/new_airlock = new path(src.loc, src)
-			new_airlock.dir = src.dir
+			path(src.loc, src)
```
Comment (PsiOmegaDelta) [inline code/game/objects/structures/door_assembly.dm]: Did you miss a new?

Comment (Dennok) [inline code/game/objects/structures/door_assembly.dm]: yes

**Comment by Zuhayr** [comment]
This needs to be squashed down into one commit and given a better commit message ('Fixes #880' would do).

**Comment by Zuhayr** [comment]
Last comment was over two weeks ago, reopen this when it has been rebased.

**Comment by Dennok** [comment]
ohh, yess. I forgot about this.

**Comment by Lorwp** [comment]
```bash
# starts interactive rebase
git rebase -i 55a137e
# commits and rebase instructions open in your set editor
```
```bash
# set commit to squash into one commit in editor
reword 55a137e add Setbounds() # change commit message to something more descriptive, like 'fixes #880'
squash 05bf886 cut werid
squash 186a626 get the dir from the assembly
squash 867ab90 return new
```
# after rebase is done, force push to branch
git push --force master
```
[More Infomation](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

**Comment by Dennok** [comment]
Thank, but, i can make this not from console in github client?

**Pull request closed without merging**